rpc/server.go

```diff
@@ -118,6 +118,24 @@ func (rs *RpcServer) registerHandlers() (err error) {
 				}
 				return l
 			})
+		case serde.GetAllLedgerChannelsMethod:
+			return processRequest(rs, requestData, func(r serde.NoPayloadRequest) []query.LedgerChannelInfo {
+				ledgers, err := rs.client.GetAllLedgerChannels()
+				if err != nil {
+					// TODO: What's the best way to handle this error?
```

Comment:
Send back an error response to the RPC client?

Reply:
Created #1275